### DISCUSSION TITLE
Documentation/Tutorial forgets to explain what a "slice" is

### DISCUSSION THREAD

**Discussion by sryze** [reduxjs/redux#4729] [Ideas]
I'm reading through the docs at https://redux.js.org/introduction/getting-started and noticed that the getting started tutorials refer to some slices and jump straight to telling me how to create a slice but there has been no definition of this term given so far. I can find it elsewhere of course, but it would be cool of the term was introduced earlier to make the learning easier. Or maybe I missed it?

**Comment by sryze** [comment]
https://redux.js.org/tutorials/essentials/part-2-app-structure#redux-slices

**Reply by markerikson** [reply]
Yeah, the "Getting Started" page is not meant to be a comprehensive intro - it's a minimal "this is what the code looks like" sample. We expect learners to go through that "Redux Essentials" tutorial, which covers all the relevant terms and concepts in detail.
